src/CMakeLists.txt

```diff
@@ -58,8 +58,10 @@ if(USE_LV2 AND USE_GUI)
     set(LV2DIR "${CMAKE_INSTALL_PREFIX}/lib/lv2/calf.lv2")
     add_library(calflv2gui SHARED gui.cpp gui_config.cpp gui_controls.cpp ctl_curve.cpp ctl_keyboard.cpp ctl_knob.cpp ctl_led.cpp ctl_tube.cpp ctl_vumeter.cpp ctl_frame.cpp ctl_fader.cpp ctl_buttons.cpp ctl_notebook.cpp ctl_meterscale.cpp ctl_combobox.cpp ctl_tuner.cpp ctl_phasegraph.cpp ctl_pattern.cpp metadata.cpp giface.cpp plugin_gui_window.cpp preset.cpp preset_gui.cpp lv2gui.cpp osctl.cpp utils.cpp ctl_linegraph.cpp drawingutils.cpp)
     target_include_directories(calflv2gui PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
-    target_link_options(calflv2gui PRIVATE -Wl,-z,nodelete)
-    target_link_libraries(calflv2gui PRIVATE ${GTK_LIBRARIES} fluidsynth)
+    if (NOT APPLE)
```

Comment:
Interestingly, this has been added to automake in #46 .

Can we make the `NOT APPLE` a `NOT SUPPORTS_NODELETE`, so that it is not only deactivated for apple, but for all linkers that do not support it?

Reply:
Of course, let me take a quick look on that change